src/coreclr/debug/createdump/createdumpwindows.cpp

```diff
@@ -20,12 +20,12 @@ CreateDump(const char* dumpPathTemplate, int pid, const char* dumpType, MINIDUMP
     hProcess = OpenProcess(PROCESS_QUERY_INFORMATION | PROCESS_VM_READ, FALSE, pid);
     if (hProcess == NULL)
     {
-        printf_error("Invalid process id '%d' error %d\n", pid, GetLastError());
+        printf_error("Invalid process id '%d' - %s\n", pid, GetLastErrorString().c_str());
```

Comment:
Similar to before: can we have the error code in there still?

Reply:
GetLastErrorString does include the error code.